ruma-events/src/room/message.rs

```diff
@@ -85,6 +85,337 @@ impl MessageEventContent {
     pub fn notice_html(body: impl Into<String>, html_body: impl Into<String>) -> Self {
         Self::new(MessageType::Notice(NoticeMessageEventContent::html(body, html_body)))
     }
+
+    /// A constructor to create a plain text reply to a message.
+    pub fn text_reply_plain(reply: impl Into<String>, original_message: &MessageEvent) -> Self {
+        let quoted = MessageEventContent::get_plain_quote_fallback(original_message);
+
+        let body = format!("{}\n\n{}", quoted, reply.into());
+
+        let text_message_content = TextMessageEventContent { body, formatted: None };
+
+        Self {
+            msgtype: MessageType::Text(text_message_content),
+            relates_to: Some(Relation::Reply {
+                in_reply_to: InReplyTo { event_id: original_message.event_id.clone() },
+            }),
+            #[cfg(feature = "unstable-pre-spec")]
+            new_content: None,
+        }
```

Comment:
This could be simplified, right?

```suggestion
        Self {
            relates_to: Some(Relation::Reply {
                in_reply_to: InReplyTo { event_id: original_message.event_id.clone() },
            }),
            ..Self::text_plain(body)
        }
```

and likewise for the other new constructors

Reply:
Thank you for the reminder of this. I simplified the 4 constructors.